frontend/src/metabase/core/components/ColorPicker/ColorPickerControls.tsx

```diff
@@ -18,7 +19,7 @@ const ColorPickerControls = CustomPicker(function ColorControls(
   props: CustomPickerInjectedProps,
 ) {
   return (
-    <div>
+    <ControlsRoot>
```

Comment:
nit: What do you think about making the name clearer and more consistent? We use the name of the component followed by `Root` in many components and I think that could be a way to go e.g. `ColorPickerControlsRoot`.